src/platform/Linux/DeviceAttestationCredsTrusty.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+ *    Copyright (c) 2021-2022 Project CHIP Authors
+ *    Copyright 2023 NXP
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#include "DeviceAttestationCredsTrusty.h"
+
+#include <crypto/CHIPCryptoPAL.h>
+#include <lib/core/CHIPError.h>
+#include <lib/support/Span.h>
+#include <trusty_matter.h>
+
+namespace chip {
+namespace Credentials {
+namespace Trusty {
+
+CHIP_ERROR TrustyDACProvider::GetDeviceAttestationCert(MutableByteSpan & out_dac_buffer)
+{
+    size_t out_size = 0;
+    int rc;
+
+    rc = trusty_matter.ExportDACCert(out_dac_buffer.data(), out_dac_buffer.size(), out_size);
+    if (rc == 0)
+    {
+        out_dac_buffer.reduce_size(out_size);
+        return CHIP_NO_ERROR;
+    }
+    else
+        return CHIP_ERROR_CERT_LOAD_FAILED;
+}
+
+CHIP_ERROR TrustyDACProvider::GetProductAttestationIntermediateCert(MutableByteSpan & out_pai_buffer)
+{
+    size_t out_size = 0;
+    int rc;
+
+    rc = trusty_matter.ExportPAICert(out_pai_buffer.data(), out_pai_buffer.size(), out_size);
+    if (rc == 0)
+    {
+        out_pai_buffer.reduce_size(out_size);
+        return CHIP_NO_ERROR;
+    }
+    else
+        return CHIP_ERROR_CERT_LOAD_FAILED;
```

Comment:
```suggestion
    return CHIP_ERROR_CERT_LOAD_FAILED;
```

Same here. Please apply throughout. Consider running clang-tidy on these files